TAG-S&P-questionnaire.md

```diff
@@ -0,0 +1,106 @@
+# Self-Review Questionnaire: Security and Privacy
+
+Questions from w3.org/TR/security-privacy-questionnaire
+
+## 2. Questions to Consider
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+If unpartitioned third-party cookies are still in use, this feature does not expose any additional information that is not already available if sites were already using an unpartitioned cross-site cookie.
+If unpartitioned third-party cookies are blocked, this provides sites in third-party contexts the ability to use HTTP cookies that are scoped to a single top-level site.
```

Comment:
"the ability to use HTTP cookies" doesn't really answer the question, unless you already know what information a cookie exposes. Maybe:

```suggestion
If unpartitioned third-party cookies are blocked, this provides sites in third-party contexts the ability to save information across visits, within the scope of a single top-level site.
```

We should also answer the "for what purposes" question. It's to track sessions, right?

Reply:
Done.